From 34c4306c3e043f7934553731ef3a4a7ab3bc7480 Mon Sep 17 00:00:00 2001
From: Zhong Huiping <huipingdx.zhong@intel.com>
Date: Fri, 25 May 2018 03:53:31 +0800
Subject: [PATCH 1434/2328] videodev2: clean up extra marco

[Change]
remove V4L2_EVENT_FRAME_END define, it's used in isys tpg
previous, but now it's not used.

Signed-off-by: Zhong Huiping <huipingdx.zhong@intel.com>
---
 include/uapi/linux/videodev2.h | 2 --
 1 file changed, 2 deletions(-)

diff --git a/include/uapi/linux/videodev2.h b/include/uapi/linux/videodev2.h
index 943123f..ffbcb25 100644
--- a/include/uapi/linux/videodev2.h
+++ b/include/uapi/linux/videodev2.h
@@ -2196,7 +2196,6 @@ struct v4l2_streamparm {
 #define V4L2_EVENT_FRAME_SYNC			4
 #define V4L2_EVENT_SOURCE_CHANGE		5
 #define V4L2_EVENT_MOTION_DET			6
-#define V4L2_EVENT_FRAME_END			7
 #define V4L2_EVENT_PRIVATE_START		0x08000000
 
 /* Payload for V4L2_EVENT_VSYNC */
@@ -2224,7 +2223,6 @@ struct v4l2_event_ctrl {
 	__s32 default_value;
 };
 
-/* V4L2_EVENT_FRAME_SYNC or V4L2_EVENT_FRAME_END */
 struct v4l2_event_frame_sync {
 	__u32 frame_sequence;
 };
-- 
2.7.4

